packages/cli/api-importers/v2-importer-commons/src/converters/schema/SchemaConverter.ts

```
@@ -307,40 +306,93 @@ export class SchemaConverter extends AbstractConverter<AbstractConverterContext<
     }: {
         context: AbstractConverterContext<object>;
         errorCollector: ErrorCollector;
-    }): Promise<Record<string, unknown>> {
-        // OAS 3.1 allows schema & property-level examples as an array of example values. See:
-        // https://stackoverflow.com/questions/73714802/multiple-examples-for-object-properties-swagger
-        const schemaExample = this.schema.example;
-        const schemaExamples = this.schema.examples;
-        let examples: unknown[] = [];
-        if (schemaExample != null) {
-            examples = [schemaExample];
+    }): Promise<FernIr.V2SchemaExamples> {
+        const v2Examples = {
+            userSpecifiedExamples: {},
+            autogeneratedExamples: {}
+        };
+
+        const examples = this.getExamplesFromSchema(errorCollector);
+
+        if (examples.length === 0) {
+            const convertedExample = await this.generateOrValidateExample({
+                shouldCollectErrors: false,
```

Why do we want to ignore the errors here? I see that we're passing in an `undefined` example value, but is it possible to refactor this so that the caller never has to decide whether or not they collect the errors?

---

The idea here is to fully autogenerate the example when provided an `undefined` example, and when doing so, we wouldn't want to log errors that come up (since we know `undefined` is a faulty example).